src/library_opfs.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2013 The Emscripten Authors
```

Comment:
Copyrights should all have the current year.

Reply:
Should this be all files that I modify or only files that I create in the current year?

Reply:
Only the year in which the file was first created. No need to update older years on files you modify.